src/primitives/cuboid.js

```diff
@@ -104,14 +103,20 @@ export default function ({ color = [1, 1, 1], coords, edgeForm = {}, opacity = 1
 
 	const cuboids = new Mesh(
 		cuboidGeometry,
-		new ShaderMaterial({
+		new RawShaderMaterial({
 			transparent: opacity !== 1,
 			depthWrite: opacity === 1,
-			lights: true,
-			uniforms: UniformsLib.lights,
-			vertexShader: `
+			uniforms,
+			vertexShader: `#version 300 es
```

Comment:
What does the comment `#version 300 es` mean? I googled and see that it probably refers to a ECMAScript version. But 300? It isn't mentioned anywhere. 

Does it mean a particular version is needed? If so why and what happens when you don't have that version? 

If it is not required, what happens then?

Reply:
> What does the comment #version 300 es mean?

That is the GLSL version, every project that uses WebGL2 uses version 3.0 ES (aka, 300).

> I googled and see that it probably refers to a ECMAScript version.

No, it is the GLSL version.

> Does it mean a particular version is needed?

Well, yes.

> If so why and what happens when you don't have that version?

Every browser that is compatible with WebGL2 is compatible with GLSL 3.0 ES, when you don't have that version, nothing but an error in the console is shown. That is the common behavior as [every modern browser supports WebGL2](https://caniuse.com/webgl2).

> If it is not required, what happens then?

It isn't required, but WebGL2 is (three.js and we use a lot of stuff that WebGL1 doesn't have) and WebGL comes with GLSL 3.0 ES, which has some nice stuff that older versions don't have.

@rocky I guess I should add a "for developers" page explaining a bit of WebGL, what do you think?

Reply:
Yes, it would be a shame if the information above were only found here in github comments where no one other than us is likely to ever see.